pkg/controller/jobframework/integrationmanager.go

```diff
@@ -101,6 +101,8 @@ func (m *integrationManager) register(name string, cb IntegrationCallbacks) erro
 	return nil
 }
 
+//+kubebuilder:rbac:groups=workload.codeflare.dev,resources=appwrappers,verbs=get;list;watch
```

Comment:
We've refrained from patching Go sources. Could these extra RBAC be added "statically" in the `rhoai` overlay instead?

Reply:
I'll take a look.  I think it could be done that way.